src/QMCWaveFunctions/WaveFunctionComponent.h

```diff
@@ -544,6 +544,13 @@ class WaveFunctionComponent : public QMCTraits
                                    std::vector<ValueType>& ratios,
                                    Matrix<ValueType>& dratios);
 
+  // Used by SOECPComponent for faster SOC evaluation
```

Comment:
Could you document this more like `evaluateDerivRatios` explaining what is evaluated?

Reply:
updated